docs/public-image-asset-guide.md

```
@@ -0,0 +1,37 @@
+# Public 이미지 자산 구조 가이드
+
+## 디렉터리 구조
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)